hddm/cnn/config.py

```diff
@@ -0,0 +1,229 @@
+import os
+import numpy as np
+import hddm
+class Config(object):
+
+    def __init__(self, model=None, bins=None, N=None):
+        # Directory setup
+        self.base_dir = hddm.__path__[0]
+        #self.data_dir = 'data'
+        #self.tfrecord_dir = 'tfrecords'
+        #self.summary_dir = 'summaries'
+
+        # training dataset features
+        self.isBinned = True
+        if model is None:
+            model = 'ddm_sdv'
+        if bins != None:
+            self.nBins = bins
```

Comment:
Everything but class names should be `snake_case`.

Reply:
ok I think I caught all of those. Convention was mostly applied.